archive/100-theorems-list/divisibility_lattice.lean

```diff
@@ -0,0 +1,159 @@
+/-
+Copyright (c) 2020 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson.
+-/
+
+import order.conditionally_complete_lattice
+import order.lattice
+import tactic.pi_instances
+
+/-
```

Comment:
```suggestion
/-!
```
Module docs need to start with `/-!`, otherwise they just become block comments and won't show up in the autogenerated docs.

Same change in the other files as well.